apps/www/styles/index.css

```diff
@@ -42,7 +43,7 @@ html {
 }
 
 body {
-  @apply bg-background;
+  @apply bg-background text-foreground;
```

Comment:
@MildTomato I just added this `text-foreground` to the body selector to fix some inheritance styling issues.

<img width="1493" alt="Screenshot 2024-02-14 at 16 35 05" src="https://github.com/supabase/supabase/assets/25671831/935ed1d7-223c-4807-8382-167e55be10c3">